binaries/data/mods/public/simulation/data/auras/units/catafalques/spart_catafalque_3.json

```diff
@@ -5,7 +5,7 @@
 	"modifications": [
 		{ "value": "ResourceGatherer/Rates/metal.ore", "multiply": 1.15 }
 	],
-	"auraName": "Iron Pelanors",
-	"auraDescription": "To further support equality, Lycurgus forbade the use of gold and silver, instead introducing new iron money called pellanors.\n+15% metal mining rate for workers within an 80 meter range.",
+	"auraName": "Iron pelanors",
+	"auraDescription": "Furthering equality measures, Lycurgus forbade the use of gold and silver, introducing instead a new iron currency called pelanors.\n+15% metal mining rate for workers within an 80 meter range.",
```

Comment:
introducing instead a new iron currency called pelanors.

I think this would sound more natural:

    introducing a new iron currency called pelanors instead.